packages/sheets/src/commands/mutations/set-worksheet-col-width.mutation.ts

```diff
@@ -68,15 +108,18 @@ export const SetWorksheetColWidthMutation: IMutation<ISetWorksheetColWidthMutati
         const defaultColumnWidth = worksheet.getConfig().defaultColumnWidth;
         const manager = worksheet.getColumnManager();
         const ranges = params.ranges;
-
-        for (let i = 0; i < ranges.length; i++) {
-            const range = ranges[i];
-            for (let j = range.startColumn; j < range.endColumn + 1; j++) {
-                const column = manager.getColumnOrCreate(j);
-                if (typeof params.colWidth === 'number') {
-                    column.w = params.colWidth;
-                } else {
-                    column.w = params.colWidth[j] ?? defaultColumnWidth;
+        if (ranges) {
```

Comment:
According to the definition of `ISetWorksheetColWidthMutationParams`, ranges can't be nil. So why do we have to do the `if` check here?